sktime/forecasting/compose/_pipeline.py

```diff
@@ -1216,6 +1216,11 @@ class ForecastX(BaseForecaster):
     If ``columns`` argument is provided, will carry ``predict`` out only for the columns
     in ``columns``, and will use other columns in ``X`` unchanged.
 
+    If variables in ``columns`` are present in the provided ``X`` during ``predict``,
```

Comment:
can we move this to the bottom of the preamble? I think the use case should be explained first.

Reply:
Moved down to just above Parameters.